the-blue-alliance-ios.xcodeproj/project.pbxproj

```diff
@@ -714,13 +726,14 @@
 				145C57BB1ED3C178004955B2 /* MatchBreakdownViewController.swift in Sources */,
 				92942DBB1E215B8D008E79CA /* MyTBAViewController.swift in Sources */,
 				924F8C871EC8C71C00F7447A /* NSManagedObjectContext+Extension.swift in Sources */,
-				924F8C8B1EC8D27B00F7447A /* RankingTableViewCell.swift in Sources */,
```

Comment:
But I'll leave this one up to you! If you want to keep them as two seperate files we can. My thought is if we want to update styles for a ranking cell we don't want to update them in two places. Ex: "Let's make the team name 1px smaller"